source/client/options_impl.cc

```diff
@@ -390,6 +406,19 @@ OptionsImpl::OptionsImpl(int argc, const char* const* argv) {
   TCLAP_SET_IF_SPECIFIED(labels, labels_);
   TCLAP_SET_IF_SPECIFIED(simple_warmup, simple_warmup_);
   TCLAP_SET_IF_SPECIFIED(no_duration, no_duration_);
+  if (stats_sinks.isSet()) {
+    envoy::config::metrics::v3::StatsSink sink;
```

Comment:
I would move this inside the `for` loop to emphasize that it's not shared across iterations.

Reply:
Done.